pkg/network.go

```diff
@@ -205,3 +218,25 @@ func (n *Network) buildP2PConfig() (*p2p.P2PNetworkConfig, error) {
 	cfg.ClientCert = cfg.ServerCert
 	return &cfg, nil
 }
+
+func (n *Network) createDocumentStore() (store.DocumentStore, error) {
```

Comment:
Shouldn't be this in /documentlog/store?

Reply:
You're right, fixed